src/main/java/org/sagebionetworks/web/client/DisplayUtils.java

```diff
@@ -669,7 +668,7 @@ public static String getSynapseHistoryTokenNoHash(
     String areaToken
   ) {
     Synapse place = new Synapse(entityId, versionNumber, area, areaToken);
-    return "!" + getPlaceString(Synapse.class) + ":" + place.toToken();
+    return "/" + getPlaceString(Synapse.class) + ":" + place.toToken();
```

Comment:
Surprised we need to add `"/"` here since it wasn't there before

Reply:
Path didn't matter before, since it was a hash fragment (it would maintain the same path).  Now the path is changing.